internal/qaengine/defaultengine_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package qaengine
+
+import (
+	"testing"
+
+	qatypes "github.com/konveyor/move2kube/types/qaengine"
+
+	"github.com/konveyor/move2kube/internal/common"
+	log "github.com/sirupsen/logrus"
+)
+
+func TestDefaultEngine(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+
+	t.Run("Fetch the default answer", func(t *testing.T) {
+
+		e := NewDefaultEngine()
+		AddEngine(e)
+
+		problem, err := qatypes.NewInputProblem("Enter the name of the registry : ",
+			[]string{"Ex : " + common.DefaultRegistryURL},
+			common.DefaultRegistryURL)
+		if err != nil {
+			log.Fatalf("Unable to create problem : %s", err)
+		}
+
+		problem, err = FetchAnswer(problem)
+		if err != nil {
+			log.Fatalf("Unable to fetch answer : %s", err)
+		}
+
+		if problem.Solution.Answer[0] != common.DefaultRegistryURL {
```

Comment:
Use the `GetXAnswer` functions:
https://github.com/konveyor/move2kube/blob/93ef9f7213a2c492c440bae2b38ba5d6e4ed46bf/types/qaengine/problem.go#L104-L145

Example:
https://github.com/konveyor/move2kube/blob/master/internal/customizer/registrycustomizer.go#L110-L113